docs/pages/guides/adding-delegation.mdx

```diff
@@ -0,0 +1,726 @@
+import { CollapseCode } from "../../components/CollapseCode";
+
+# Adding delegation
+
+On this page you learn how to add [delegations](https://mud.dev/world/account-delegation) to enable addresses to act on behalf of other addresses.
+This is important, for example, because typically every game move is a transaction, but you wouldn't want the player to constantly have to confirm every move on the wallet extension.
+The common pattern is to create a burner address and have the wallet user address authorize it to play on its behalf.
+
+## The sample application
+
+The application is located [in our monorepository](https://github.com/latticexyz/mud/tree/main/examples/multiple-accounts).
+It creates multiple burner addresses, and keeps track onchain when they last submitted a transaction.
+
+There are the steps to run the initial application.
+
+1. Clone the monorepo, move the application out, and delete the monorepo again.
```

Comment:
Why not just `cd mud/examples/multiple-accounts`?

Reply:
I just didn't think it was necessary to keep the entire repository. But you're right - I forgot disc space is a practically unlimited resource.